### PR TITLE
change to protected a/b/c for security class

### PR DESCRIPTION
**Pull request by BrandonSharratt**
For reference see https://github.com/bcgov/ckan-ui/issues/144

### CODE REVIEW

File: scripts/scheming_migration/index.js [hunk 1]
```diff
@@ -334,16 +365,16 @@ async function main() {
...
-				JSON.stringify(packageExtras['contacts']),
-				JSON.stringify(packageExtras['dates']),
-				JSON.stringify(packageExtras['more_info']),
-				packageExtras['edc_state'],
-				packageObj['revision_id'],
-				uuidv4(),
-				uuidv4(),
-				uuidv4(),
-				uuidv4()
+				packageObj['id'], //1
```
Comment (SteveChapmanBCDX) [inline scripts/scheming_migration/index.js:368]: Are these comments?  Did you mean to add them?

**Review by SteveChapmanBCDX — APPROVED** [review]
@BrandonSharratt - These changes look fine, subject to my question about comments.

File: scripts/scheming_migration/index.js (hunk 1 above)
Comment (BrandonSharratt) [reply scripts/scheming_migration/index.js:368]: yes I meant to add them, I initially tried to do the change as part of that query and I found it impossible to read without the number there that corresponds to the number it is in the query above.

Comment (SteveChapmanBCDX) [reply scripts/scheming_migration/index.js:368]: Thanks - that makes sense.

**Pull request merged**